.pipelines/templates/e2e-test-kind.yaml

```diff
@@ -39,6 +39,16 @@ jobs:
           IS_HELM_TEST: false
 
     steps:
+      # logging in to download the sa.pub and sa.key used for creating the kind cluster
+      # with OIDC issuer enabled
+      - script: |
+          az login --service-principal -u $AZURE_CLIENT_ID -p $AZURE_CLIENT_SECRET --tenant $AZURE_TENANT_ID --allow-no-subscriptions > /dev/null
+          az account set --subscription $AZURE_SUBSCRIPTION_ID > /dev/null
```

Comment:
Using the azure sp we used for the e2e tests for login here because it has minimal permissions on the keyvault. This login is for getting the signing keys used for bootstrapping the kind cluster.